### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by eliteProgrammer-1**
Created Login Functionality.

### CODE REVIEW

File: Development/Web Content/HTML/loginPage.html [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
     <div class="container">
-        <form action="/" method="post">
+        <form action="login-verification" method="post">
             <h3>Login</h3>
-            <input type="text" placeholder="Employee Id" class="inp">
-            <input type="password" placeholder="Password" class="inp">
-            <select name="user_type" id="" class="inp user_type">
+            <input type="text" placeholder="Employee Id" class="inp" name="emp_id">
+            <input type="password" placeholder="Password" class="inp" name="password">
+            <!-- <select name="user_type" id="" class="inp user_type">
```
Comment (himanshu-gupta-intimetec) [inline Development/Web Content/HTML/loginPage.html]: Remove commented code if not needed in near future

File: Development/src/login/User.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import javax.servlet.http.*;
+
+public class User extends HttpServlet
+{
+    private String emp_id, password;
+    private boolean isSecurityPersonnel, isHR, isEmployee;
+    
+    public User()
+    {
+        this.emp_id = "";
```
Comment (himanshu-gupta-intimetec) [inline Development/src/login/User.java]: Create seperate string constants file and store empty string there and then use it

File: Development/src/login/User.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        this.password = "";
+        isSecurityPersonnel = false;
+        isEmployee = false;
+        isHR = false;
+    }
+    
+    public void setUserType(String user_type)
+    {
+    	if((user_type.toLowerCase()).equals("hr"))
```
Comment (himanshu-gupta-intimetec) [inline Development/src/login/User.java]: remove string constants

File: Development/src/login/User.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    
+    public void redirectAccordingToUserType()
+    {
+    	
+    }
+    
+    
+    public boolean isAuthenticated(String emp_id, String password)
+    {   
+        String databaseURL = "jdbc:mysql://localhost:3306/namaste_visitor";
```
Comment (himanshu-gupta-intimetec) [inline Development/src/login/User.java]: store database related constants and path in seperatae file

File: Development/src/login/User.java [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+            ResultSet resultSet = preparedStatement.executeQuery();
+            if(resultSet.next() == true)
+            {	
+            	String user_type = resultSet.getString("user_type");
+            	setUserType(user_type);
+                return true;
+            }
+        }
+        catch(Exception e)
```
Comment (himanshu-gupta-intimetec) [inline Development/src/login/User.java]: What is the sense of using catch here if you are not doing anything here. Handle this properly

File: Development/src/login/User.java (hunk 4 above)
Comment (eliteProgrammer-1) [reply Development/src/login/User.java]: Done

File: Development/src/login/User.java (hunk 5 above)
Comment (eliteProgrammer-1) [reply Development/src/login/User.java]: Done

File: Development/src/login/User.java (hunk 2 above)
Comment (eliteProgrammer-1) [reply Development/src/login/User.java]: made changes.

File: Development/src/login/User.java (hunk 3 above)
Comment (eliteProgrammer-1) [reply Development/src/login/User.java]: created constant variables in separate file.

**Pull request merged**